src/compiler/diagnosticMessages.json

```diff
@@ -3386,6 +3386,10 @@
         "category": "Error",
         "code": 4110
     },
+    "Access property '{0}' by index signature is disallowed.": {

```

Comment:
Property '{0}' comes from an index signature, so it must be accessed with ["{0}"].
OR
"... must be accessed with element access."